Origam.PrintServer/Origam.PrintServer.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+<PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
how about setting the framework to net8.0-windows if the program can really only run on windows?